TACC/Core-Portal
# Added () to accepted characters and changed error message to reflect

## Overview
Added () to accepted characters and changed error message to reflect


## Related

* [FP-1668](https://jira.tacc.utexas.edu/browse/FP-1668)

## Changes

Rename modal now allows the use of parentheses "()" and the error message will include that now as an acceptable character.

## Testing

1. Go to My Dashboard >> Data Files
2. Select a Workspace from the Data Files Nav (Public Data, Shared Workspace, etc)
3. Click checkbox of data file you'd like to edit
4. Click Rename
5. When modal pops up enter a name with a invalid character like &%$ to make sure the error message says () are ok
6. When modal pops up, enter a name that includes parentheses and make sure it successfully saves

## UI



## Notes
